docs/book/aya/crosscompile.md

```diff
@@ -0,0 +1,49 @@
+# Cross-compiling aya-based programs
+
+The instructions below show how to cross compile aya eBPF programs on Macs.
+Cross compiling on other systems is possible too and we're going
+to add instructions to do that soon (PRs welcome!).
+
+# Cross-compiling aya-based programs on Mac
+
+Cross compilation should work on both Intel and Apple Silicon Macs.
+
+1. Install `rustup` following the instructions on <https://rustup.rs/>
+1. Install a rust stable and nightly toolchains:
```

Comment:
Install the stable and nightly rust toolchains

Reply:
changed in [a6946fe](https://github.com/aya-rs/book/pull/132/commits/a6946fefb79d221034277e2359ee88d5c24e3fc9)